client/container.go

```diff
@@ -570,26 +477,16 @@ func (c *Client) ContainerEACL(ctx context.Context, prm PrmContainerEACL) (*ResC
 	return &res, nil
 }
 
-// PrmContainerSetEACL groups parameters of ContainerSetEACL operation.
+// PrmContainerSetEACL groups optional parameters of ContainerSetEACL operation.
 type PrmContainerSetEACL struct {
 	prmCommonMeta
 
-	tableSet bool
-	table    eacl.Table
-
 	sessionSet bool
 	session    session.Container
 
 	signer neofscrypto.Signer
 }
 
-// SetTable sets eACL table structure to be set for the container.
-// Required parameter and CID must be set inside the table.
```

Comment:
`CID must be set inside the table` requirement must be kept

Reply:
Moved this to command description